server/src/com/thoughtworks/go/server/view/artifacts/ArtifactDirectoryChooser.java

```diff
@@ -75,7 +75,7 @@ public File findCachedArtifact(LocatableEntity locatableEntity) {
     }
 
     public File temporaryConsoleFile(LocatableEntity locatableEntity) {
-        return new File("work/local", format("%s.log", DigestUtils.md5Hex(locatableEntity.entityLocator())));
+        return new File("data/console", format("%s.log", DigestUtils.md5Hex(locatableEntity.entityLocator())));
```

Comment:
Yes at https://github.com/gocd/gocd/blob/master/server/src/com/thoughtworks/go/server/service/ConsoleService.java#L115

and

https://github.com/gocd/gocd/blob/master/server/src/com/thoughtworks/go/server/service/ConsoleService.java#L178